### PR TITLE
fix(app, components): center content in instruments and modules cards

### PR DESCRIPTION
**Pull request by ncdiehl11**
closes [RQA-1488](https://opentrons.atlassian.net/browse/RQA-1488)

# Overview

Instrument, Pipette, and Module Cards shown in InstrumentsAndModules are all styled differently. This PR aligns content on each card for both Flex and OT-2. It may be worthwhile in the future to refactor gripper, pipette, and module cards to an abstract Card component with an image and text content.

# Test Plan

- Check that Flex gripper, pipette, and module card content are aligned 
Screen Shot 2023-10-03 at 10 21 59 AM

- Check that OT-2 pipette and module card content are aligned 
Screen Shot 2023-10-03 at 10 21 34 AM

# Changelog

- Change padding and image sizing on instrument, pipette, and module cards

# Risk assessment

low

[RQA-1488]: https://opentrons.atlassian.net/browse/RQA-1488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/molecules/InstrumentCard/index.tsx [hunk 1]
```diff
@@ -69,8 +69,18 @@ export function InstrumentCard(props: InstrumentCardProps): JSX.Element {
       {...styleProps}
     >
       {isGripperAttached ? (
-        <Flex justifyContent={JUSTIFY_CENTER} size="3.75rem">
-          <img src={flexGripper} alt="flex gripper" />
+        <Flex
+          justifyContent={JUSTIFY_CENTER}
+          size="3.75rem"
+          width={'60px'}
+          height={'54px'}
```
Comment (jerader) [inline app/src/molecules/InstrumentCard/index.tsx]: the size is unnecessary now. also, maybe convert the pixels into rem? We typically try to use rem unless you are sizing an img html tag

```suggestion
          width="60px"
          height="54px"
```

File: app/src/molecules/InstrumentCard/index.tsx [hunk 2]
```diff
@@ -69,8 +69,18 @@ export function InstrumentCard(props: InstrumentCardProps): JSX.Element {
...
+        >
+          <img
+            src={flexGripper}
+            alt="flex gripper"
```
Comment (jerader) [inline app/src/molecules/InstrumentCard/index.tsx]:
```suggestion
            alt="Flex Gripper"
```

File: components/src/instrument/InstrumentDiagram.tsx [hunk 3]
```diff
@@ -40,13 +40,19 @@ export function InstrumentDiagram(props: InstrumentDiagramProps): JSX.Element {
...
   }
   return (
-    <Box
-      className={className}
+    <Flex
       transform={mount === 'right' ? 'scaleX(-1)' : ''}
       filter={mount === 'right' ? 'FlipH' : ''}
-      {...styleProps}
+      width="60px"
+      height="54px"
```
Comment (jerader) [inline components/src/instrument/InstrumentDiagram.tsx]: nit changing these to rem too

**Comment by jerader** [comment]
Also, `InstrumentDiagram` is a shared component with PD. Looks like the changes affected both OT-2 and Flex pipette images in the Pipettes edit section.

Screen Shot 2023-10-03 at 10 42 22 AM

File: components/src/instrument/InstrumentDiagram.tsx (hunk 3 above)
Comment (ncdiehl11) [reply components/src/instrument/InstrumentDiagram.tsx]: will do. also, I missed that the InstrumentDiagram component is shared with PD. I'll resolve this

File: components/src/instrument/InstrumentDiagram.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { FlattenSimpleInterpolation } from 'styled-components'
-import { Box } from '..'
+import { Flex, ALIGN_CENTER, JUSTIFY_CENTER } from '..'
```
Comment (jerader) [inline components/src/instrument/InstrumentDiagram.tsx]: nit: lets specify the exact import location for all of these so we don't run into circular dependency issues down the road.

```suggestion
import { Flex } from '../primitives'
import { ALIGN_CENTER, JUSTIFY_CENTER } from '../styles'
```

**Review by jerader — APPROVED** [review]
Lgtm! PD looks better - thank you! The pipette modal's left pipette looks a bit off but its from a parent component's styling and we can fix later. Also i left a small comment about imports.

File: app/src/organisms/Devices/PipetteCard/index.tsx [hunk 5]
```diff
@@ -194,19 +195,17 @@ export const PipetteCard = (props: PipetteCardProps): JSX.Element => {
       {!pipetteIsBad && subsystemUpdateData == null && (
         <>
           <Box
-            padding={`${SPACING.spacing16} ${SPACING.spacing8}`}
+            padding={`${SPACING.spacing16} ${SPACING.spacing16}`}
```
Comment (koji) [inline app/src/organisms/Devices/PipetteCard/index.tsx]:
```suggestion
            padding={SPACING.spacing16}
```

File: app/src/organisms/ModuleCard/index.tsx [hunk 6]
```diff
@@ -279,7 +279,7 @@ export const ModuleCard = (props: ModuleCardProps): JSX.Element | null => {
           onCloseClick={() => setShowTestShake(false)}
         />
       )}
-      <Box padding={`${SPACING.spacing16} ${SPACING.spacing8}`} width="100%">
+      <Box padding={`${SPACING.spacing16} ${SPACING.spacing16}`} width="100%">
```
Comment (koji) [inline app/src/organisms/ModuleCard/index.tsx]:
```suggestion
      <Box padding={SPACING.spacing16} width="100%">
```

**Pull request merged**